recipe/meta.yaml

```diff
@@ -145,7 +145,7 @@ outputs:
         - healpy >=1.16.6
         - healsparse >=1.8.1
         - hpgeom >=1.2.0
-        - htcondor >=23.0
+        - htcondor >=24.0
```

Comment:
I think formally we want this to be `>=24.0,<24.1` as I noted on Slack. Because the x.0.y series are the LTS versions.